include/siminusminus/containers/inmutablestring.hpp

```diff
@@ -59,14 +64,15 @@ class InmutableString
      * Returns the i th position from our string.
      * @param index: index to string access.
      */
-    char operator[](const size_t index) const;
+    char& operator[](const size_t index) const;
```

Comment:
No, this change is wrong (And should not be compiling at all). It was ok in the previous version